app/views/common/_multiple_input.html.erb

```diff
@@ -24,7 +24,7 @@ render :partial => 'common/multiple_input', :locals => {:field_name => 'keywords
 <% classes += ' hidden' if hidden %>
 <% classes += ' has-error' if errors.size > 0 %>
 
-<div class=<%= classes %> title='<%= title %>'>
+<div class=<%= classes %> title='<%= title %>' id=<%= "multi_input_#{field_name}" %>>
```

Comment:
Even though most browser probably tolerate this, it should have quotes around the ERB expressions e.g. `id='<%=...`, also for the existing `class=<%=...` attribute

Reply:
Tests are breaking when I also change class so I'll keep it to id for this PR